<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inputwindow</class>
 <widget class="QWidget" name="Inputwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Inputwindow{
	background: rgb(188,188,188);
}
#infoLabel{
	color: rgb(68, 68, 68);
	font: bold 28px;
	border: 0px;
	padding: 0px;
}
QLabel{
	font: bold 22px;
	color: rgb(68, 68, 68);
	border: 2px solid rgb(68, 68, 68);
	padding: 2px;
}
QPushButton{
	outline : 0px;
	font: bold 14px;
	color: white;
	background: rgb(68, 68, 68);
	border-radius: 7px;
}

QPushButton:pressed{ 
	background:rgb(134, 134, 134) ;
}
QCheckBox{
	outline: 0px;
}
QCheckBox::indicator{
	width: 50px;
	height: 50px;
}
QCheckBox::indicator:checked{
	image:url(&quot;:/Button/switchOn512x272.png&quot;);
}
QCheckBox::indicator:unchecked{
	image:url(&quot;:/Button/switchOff512x272.png&quot;);
}

QSpinBox{
	background: rgb(188,188,188);
	font: bold 20px;
	color: rgb(68, 68, 68);
	border: 2px solid rgb(68, 68, 68);
}
QSpinBox::up-button{
	font: bold 50px;
}</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>894</x>
     <y>510</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>30</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PLC INPUT TEST</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="switchInput0_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>56</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_0Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>106</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_1Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>156</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_2Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>206</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_3Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>256</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_4Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>306</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.4</string>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_5Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>356</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_6Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>406</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.6</string>
   </property>
  </widget>
  <widget class="QLabel" name="Input0_7Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>456</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>I 0.7</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputB1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>-128</number>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputB2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>63</width>
     <height>34</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputW3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>-32768</number>
   </property>
   <property name="maximum">
    <number>32768</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputW5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
